benchmark/benchmark.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
```

Comment:
could you add a `README` under `benchmark` so maintainers have more info on how and when to run it?

Reply:
Will do